docs/user/user/metrics/avoiding-the-environment.md

```diff
@@ -0,0 +1,39 @@
+# FIXME(title): Avoiding the Environment
+
+The Glean SDK collects a limited amount of metrics that are generally useful across products.
+These metrics are sent in the [`client_info`][client_info] section of every ping.
+The data is provided by the embedding application or automatically fetched by the Glean SDK.
+It is collected at initialization time and sent in every ping afterwards.
+For historical reasons it contains metrics that are only useful on a certain platform.
+
+[client_info]: ../pings/index.md#the-client_info-section
+
+## Adding new metrics to `client_info`
+
+Adding new metrics maintained by the Glean SDK team will require a full proposal
+and details on why that value is useful across multiple platforms and products and needs SDK team ownership.
```

Comment:
At the very least this needs a bug so we add some notes what the proposal should do.